pdata/pcommon/spanid.go

```diff
@@ -20,37 +20,35 @@ import (
 )
 
 // EmptySpanID represents the empty (all zero bytes) SpanID.
-var EmptySpanID = NewSpanID([8]byte{})
+var EmptySpanID = SpanID([8]byte{})
```

Comment:
I think SpanID and TraceID now can be treated the same way as byte arrays. In that case we are exposing the individual bytes for possible mutation like `EmptySpanID[1] = 123`. I believe it's applicable to any instance of SpanID and SpanID now

Reply:
>  I believe it's applicable to any instance of SpanID and TraceID now

In Go language, an array is of value type not of reference type. So when the array is assigned to a new variable, then the changes made in the new variable do not affect the original array.

So the returned SpanID/TraceID can indeed be mutated, see also the tests that proves the initial array is not mutated by changing the return value type array (not a reference to the initial array).

The problem with `EmptySpanID` is valid though, so need to change. Is `NewTraceIDEmpty` a good name?

Reply:
Maybe `NewEmptyTraceID`?

Reply:
I would keep it consistent with `NewValueEmpty`?

Reply:
Ok, let's keep `NewValueEmpty`